### ISSUE TITLE
Session cannot be set since upgrade to FrankenPHP

### ISSUE THREAD

**Issue by HansPeterOrding** [dunglas/symfony-docker#470]
I tried to implement a User Login form from scratch following symfony documentation but I never got logged in properly. Debugging lead to the following results:
* I don't have a symfony session id ($requestStack->getSession()->getId() is always null
* I can see a session id in my browsers dev toolbar (Application Tab)
* The User provider properly loads the user but a call to $this->getUser() in any controller always gives null

What i can tell besides:
* The /var folder is not syncronized to local
* If I switch to the session native file session handler, the directory /var/sessions/dev is created but no file is stored
* I have an insecure connection but setting cookie secure to false does not help anyways

I would appreciate any idea what could cause this behaviour. Unfortunately I suspect something with the newly introduced FrankenPHP setup. The exact same approach always worked nicely before.

**Comment by HansPeterOrding** [comment]
Further information:
I just rolled back to b5710da39cc9939c2eef4787ab50b4ee7d16e44f (last commit without FrankenPHP) and everything works like a charm.
There must be some issue in the new version with session storage.

**Comment by traoreoumar** [comment]
I think it is because of the `variables_order = EGPS` line in frankenphp/conf.d/app.ini.
I commented this line and it works.

**Comment by dunglas** [comment]
Good catch @traoreoumar!

I was able to reproduce with this simple script:


```php
<?php

namespace App\Controller;

use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
use Symfony\Component\HttpFoundation\RequestStack;
use Symfony\Component\HttpFoundation\Response;
use Symfony\Component\Routing\Annotation\Route;

class SessionController extends AbstractController
{
    #[Route('/', name: 'app_session')]
    public function index(RequestStack $requestStack): Response
    {
        $session = $requestStack->getSession();

        $i = $session->get('counter', -1);
        $session->set('counter', ++$i);

        return $this->json(['counter' => $i]);
    }
}
```

Changing `variables_order` to `EGPCS` fixes the issue: #471.

(For the record, [session support is tested in FrankenPHP](https://github.com/dunglas/frankenphp/blob/main/frankenphp_test.go#L304).
